src/hooks/useConcepts.tsx

```diff
@@ -39,7 +39,10 @@ export function useConcepts(references: Set<string>): {
 
   const results = useMemo(
     () => ({
-      concepts: data ? [].concat(data?.map((res) => res?.data?.results).flat()) : undefined,
+      // data?.[0] check is added for tests, as response is undefined in tests
+      // hence the returned concepts are [undefined], which breaks the form-helper.ts
```

Comment:
Makes sense @vasharma05 but since we don't expect such issues in production, does it make sense to fix this in the test env instead? Like maybe mocking the `useSWRInfinite` or its wrapper to behave as expected?

Reply:
Actually, the present mocks have added the conditions for `openmrsFetch` instead of `useConcepts`. Adding the mock for `useConcepts` will then break those mocks.